scaletest/createworkspaces/run.go

```diff
@@ -176,13 +176,14 @@ resourceLoop:
 }
 
 // Cleanup implements Cleanable.
-func (r *Runner) Cleanup(ctx context.Context, id string) error {
+func (r *Runner) Cleanup(ctx context.Context, id string, w io.Writer) error {
```

Comment:
I like the name `logs` as clearer intention than `w`

Reply:
Renamed